api/controllers/search.js

```diff
@@ -86,7 +87,17 @@ const queryFileFromReq = req => {
 
 const offsetFromReq = req => {
   const page = as.number(req.query.page || 1);
-  return (page - 1) * RESPONSE_LIMIT;
+  const limit = limitFromReq(req);
```

Comment:
this line seems useless?

Reply:
Oh yes, lost my place mid-refactor. Good catch.